### PR TITLE
Introducing --silent and --sourcemap flags.

### PR DESCRIPTION
**Pull request by sznowicki**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -159,7 +160,20 @@ const cacheEntry = (isStaging || isProd) ? [
...
+/**
+ * Gets the source map mode which should be used within
+ * the webpack config.
+ * @return {string}
+ */
+const getSourceMapType = () => {
+  if (sourcemap) {
+    return sourcemap;
+  }
+  if (isDev) {
```
Comment (devbucket) [inline webpack.config.js]: Codestyle: Please add a blank line before condition.

File: webpack.config.js [hunk 2]
```diff
@@ -159,7 +160,20 @@ const cacheEntry = (isStaging || isProd) ? [
...
+    return 'eval';
+  }
+  return 'source-map';
```
Comment (devbucket) [inline webpack.config.js]: Codestyle: Please add a blank line before condition.

File: webpack.config.js [hunk 3]
```diff
@@ -159,7 +160,20 @@ const cacheEntry = (isStaging || isProd) ? [
 const performance = (isDev) ? [
   resolve(__dirname, 'modules', 'performance'),
 ] : [];
-
+/**
+ * Gets the source map mode which should be used within
+ * the webpack config.
+ * @return {string}
+ */
+const getSourceMapType = () => {
```
Comment (devbucket) [inline webpack.config.js]: Please move this function to the webpack helpers. `./helpers/webpack/getSourceMapType.js` (please follow the "new" principle of separate files and ignore the index.js

**Comment by devbucket** [comment]
Please point the pull request against `development`, not `master`!

**Comment by sznowicki** [comment]
@devbucket isn't master available via npm?

**Comment by devbucket** [comment]
@sznowicki Yes, the master is available via npm only. But we always merge PRs into development first ... for testing and CI purposes. And then we create a new version and create a release from the master.

File: helpers/webpack/getSourceMapType.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ * the webpack config.
+ * @return {string}
+ */
+module.exports = () => {
+  if (sourcemap) {
+    return sourcemap;
+  }
+
+  if (isDev) {
+    return 'cheap-eval-source-map';
```
Comment (devbucket) [inline helpers/webpack/getSourceMapType.js]: Please compare with the current version of the devserver. The used presets have changed.

File: helpers/webpack/getSourceMapType.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  return 'cheap-source-map';
```
Comment (devbucket) [inline helpers/webpack/getSourceMapType.js]: Please compare with the current version of the devserver. The used presets have changed.

**Pull request merged**